automation/check-patch.e2e-lifecycle-k8s.sh

```diff
@@ -18,7 +18,7 @@ versionChanged() {
 }
 
 main() {
-    export KUBEVIRT_PROVIDER='k8s-1.25'
+    export KUBEVIRT_PROVIDER='k8s-1.28'
```

Comment:
we can even drop this line (and the one in the next file)
as it is the default anyhow taken from cluster.sh

Reply:
Please drop it

Reply:
done, lets see all pass